source/_integrations/plugwise.markdown

```diff
@@ -33,17 +33,21 @@ The password can be found on the bottom of your Smile, it should consist of 6 ch
 
-For example, if you have an Adam setup with a Lisa named 'Living' and a Tom named 'Bathroom' these will show up as individual devices. For setups where an auxiliary heater (or heater/cooler) is present there will be an additional device representing it called 'Auxiliary'.
+For example, if you have an Adam setup with a Lisa named 'Living' and a Tom named 'Bathroom' these will show up as individual entities. When there are more (than one) Plugwise thermostats present, an "Auxiliary Device" will be added representing the active heating/cooling device present in your climate system. If you have Plugs (as in, pluggable switches connecting to an Adam) those will be discovered as `switch`es. Various other measures of your setup will be available as `sensor`s or `binary_sensor`s.
 
-Centralized measurements such as 'power' for a P1, 'outdoor_temperature' on Anna or Smile will be assigned to your gateway device. Auxiliary Heater(/Cooler) measurements such as 'boiler_temperature' will be assigned to the Auxiliary entity.
+Centralized measurements such as 'power' for a P1, 'outdoor_temperature' on Anna or Smile will be assigned to your gateway entity. Auxiliary Heater(/Cooler) measurements such as 'boiler_temperature' will be assigned to the Auxiliary entity.
 
 ## Configuration
 
-To set up this integration, click Configuration in the sidebar and then click Integrations. Add a new integration using the "+" button in the lower right corner and look for 'Plugwise'. Click configure and you will be presented with a dialog requesting the Smile ID or password of your Smile and it's IP address. After you click submit, you will have the opportunity to select the area(s) where individual Smile appliances are located.
+The Plugwise Smile(s) present in your network will be automatically detected via Zeroconf discovery and will be shown on the Integrations-page. To set up an integration, click CONFIGURATION on the discovered Entity and you will be presented with a dialog requesting the Smile password. After you click submit, you will have the opportunity to select the area(s) where individual Smile appliances are located.
 
-Depending on your `climate` setup an auxiliary entitiy will be added when there is information available about such Plugwise devices. If you have "plug"s (as in, pluggable switches that come with an Adam) those will be discovered as `switch`es. Various other measures of your setup will be available as `sensor`s or `binary_sensor`s.
+PLEASE NOTE 1: the Unique ID(s) of the Plugwise integration(s) has(have) changed due to the introduction of automatic discovery. If you have already configured the Plugwise integration(s) via the "Integrations --> Plus-button"-method, you can ignore the newly discovered Plugwise integration(s).
+
+PLEASE NOTE 2: when you have an Anna and an Adam, make sure to only configure the Adam integration. You can press IGNORE on the Anna integration to remove this integration. 
```

Comment:
Use a note box instead of "PLEASE NOTE"

Reply:
Sorry, how do I do that?

Reply:
<https://developers.home-assistant.io/docs/documenting/create-page#html>

Reply:
I've implemented the note box, but I see no difference from the normal text?

Reply:
GitHub won't render it, but once it goes through jekyll it will render properly.

Reply:
OK clear tnx.